cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -557,6 +557,40 @@ storage:
             volumeMounts:
             - name: config-volume
               mountPath: /etc/nginx
+          - name: skipper-proxy
+            image: registry.opensource.zalan.do/pathfinder/skipper:v0.10.233
```

Comment:
TODO: Needs to be updated to a skipper that allows POST requests to do protocol upgrades, see https://github.com/zalando/skipper/pull/1091.